osu.Game/Screens/Play/DimmableStoryboard.cs

```diff
@@ -57,6 +59,15 @@ private void initializeStoryboard(bool async)
             if (!ShowStoryboard.Value && !IgnoreUserSettings.Value)
                 return;
 
+            double storyboardLastestEventTime = storyboard.LatestEventTime ?? 0;
+
+            if (storyboard.BeatmapInfo.Metadata?.BackgroundFile!=null && !storyboard.ReplacesBackground && storyboardLastestEventTime != 0)
```

Comment:
Why is `storyboardLastestEventTime` (typo) coerced to 0 and then checked against 0 again, instead of being simply null-checked?

Reply:
This is still not resolved in the newest changes.

Reply:
It's because the `LatestEventTime` is calculated twice here and it's recommended to store its value locally.

Reply:
...it still can be stored to a variable nevertheless, just *without* the coercion to 0?